tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -24,6 +24,8 @@ for allowed_hypervisor_annotation in "${non_formatted_allowed_hypervisor_annotat
 done
 allowed_hypervisor_annotations=$(echo $allowed_hypervisor_annotations | sed 's/,$//')
 
+getenforce &>/dev/null && SELINUX=${SELINUX:-yes} || SELINUX=${SELINUX:-no}
```

Comment:
Actually, now looking at this we have an issue.

`getenforce` is coming from the host, while this script is running from inside a container.
If we want to call `getenforce` we need to explicitly mount `/usr/bin/` into the container ... or use `nsenter` to be able to call the binary on the host side, which is preferred.

Reply:
Hello Fabiano, I got the impression it runs inside the host as it's modifying the `/opt/kata`, do you have a precedence in how to obtain SELINUX value? I noticed it's used elsewhere but I wasn't sure it gets set and propagated into `kata-deployment` pod which is why I tried to get a better default. Another option would be to simply add it always and use `|| true` (but I haven't seen such handling elsewhere so I chose this instead)